tests/queries/0_stateless/00626_replace_partition_from_table.sql

```diff
@@ -1,4 +1,5 @@
--- Tags: no-parallel
+-- Tags: no-parallel, no-random-detach
+-- Tag no-random-detach: it's a bug, will be fixed in https://github.com/ClickHouse/ClickHouse/pull/41145
```

Comment:
The bug is that after `TRUNCATE` or `DROP PARTITION` a new part with the same name (and path) may appear, but marks/uncompressed cache is not dropped, which leads to reading of wrong data. Most likely will be fixed after #41145.

Reply:
Is it?